src/main/java/no/fint/personalmappe/service/PolicyService.java

```diff
@@ -0,0 +1,40 @@
+package no.fint.personalmappe.service;
+
+import no.fint.model.resource.administrasjon.personal.PersonalmappeResource;
+import no.fint.personalmappe.policy.helper.LinkHelper;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+import javax.script.Invocable;
+import javax.script.ScriptEngine;
+import javax.script.ScriptEngineManager;
+import javax.script.ScriptException;
+
+@Service
+public class PolicyService {
+
+    private ScriptEngine engine;
+
+    @PostConstruct
+    public void init() {
+        engine = new ScriptEngineManager().getEngineByName("nashorn");
+        engine.put("resource", LinkHelper.resource());
+    }
+
+    public PersonalmappeResource transform(String policy, PersonalmappeResource personalmappeResource) {
+        try {
+            engine.eval(policy);
+
+            Object o = ((Invocable) engine).invokeFunction(getFunctionName(policy), personalmappeResource);
+            return (PersonalmappeResource) o;
+        } catch (ScriptException | NoSuchMethodException e) {
+            e.printStackTrace();
+        }
+
+        return null;
+    }
+
+    public String getFunctionName(String policy) {
```

Comment:
Perhaps require a standard name of the function to call instead.  A complex policy could contain several functions.

Reply:
That's a good idea.

Reply:
But I think in this version we should only support one function.

Reply:
After some thought i think we should wait with this one until we move to a Azure blob or something. The name of the function could be a good documentation of the policy if we only support one function in this release.